build/yamls/windows/containerd-with-ovs/conf/Install-OVSDriver-Containerd.ps1

```diff
@@ -0,0 +1,18 @@
+$ErrorActionPreference = "Stop"
+mkdir -force c:/openvswitch/driver
```

Comment:
No need to create this directory?

Reply:
It seems we don't have this folder in the container..

Reply:
This command is trying to create directory on the host as your are using path "C:/openvswitch"

Reply:
Got you points, will remove it, thanks.